### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by Swarzstein**
Creation of the modal for the cards and JavaScript functionality with DOM manipulation.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -1,6 +1,139 @@
...
+  }
+  document.querySelector('#card-modal').classList.toggle('hidden');
+  document.querySelector('#modal-title').textContent = project.title;
+  document.getElementById('modal-tecs').innerHTML = technologies;
+  document.querySelector('#modal-img').textContent = project.imgb;
+  document.querySelector('#modal-desc').textContent = project.description;
+}
+
+function closeModal() {
+  document.querySelector('#card-modal').classList.toggle('hidden');
```
Comment (MaganaAsiati) [inline js/main.js:133]: - Hey :hand: Here you did a great job here but make sure your project working properly as explained below when we clique to the see project button shown below  the popup appear with the detail of the project :+1: :smiley: 


image

The [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters is set correctly project repository.
When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
When the user clicks (or taps) the close (X) button, the popup disappears.
The popup is responsive, it fits mobile and desktop sizes properly.
The pop-up follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins).

File: js/main.js [hunk 2]
```diff
@@ -1,6 +1,139 @@
...
+  card6: {
+    id: 6,
+    title: 'Profesional Art Printing Data',
+    imgs: '../img/card-img-1.png',
+    imgb: '../img/Img-Placeholder.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    mProject: false,
+  },
+};
```
Comment (MaganaAsiati) [inline js/main.js:65]: - Kindly make sure you implement those points below to add the detail of each project in the pop-up :+1: 
   - The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project.
   - Each object contains at least the following information about the project: name, description, featured image, technologies, link to live version, and link to the project source code. Ie: :+1: :smiley: 


```
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

File: index.html [hunk 3]
```diff
@@ -98,7 +114,7 @@ <h2 class="title-post-2">Profesional Art Printing Data</h2>
               <li><button class="p-lang-2">Ruby</button></li>
             </ul>
           </div>
-          <button class="link-btn-2 fill-h">See Project</button>
```
Comment (MaganaAsiati) [inline index.html:101]: - Kindly make sure each project has the `see project `button in your desktop version only one project have the seen button please fix that so   when someone will click on that see project button it  will show the popup with all detail of your project :+1: 

image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 

Hi @sonyco-4u 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✅ No linter error
✅ README is professional

## Required Changes ♻️

- Kindly make sure you implement properly the functionality of the pop-up window as it require in the project I list more detail below kindly check them :+1: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @Swarzstein 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**